tests/test_model.py

```diff
@@ -76,6 +77,9 @@ def test_cpes():
     ps3node = ProductNode.objects.create(parent=pv2node, obj=ps3)
     assert ps3node
 
+    assert ps3node.name == "Brantabernicla"
+    assert ps3node.desc == "The brant or brent goose is a small goose of the genus Branta."
```

Comment:
Rather than hardcoding a value, I think it's better to check that the ProductNode object has its name and description copied from the ProductStream object like we expect:
```suggestion
    assert ps3node.name == ps3.name
    assert ps3node.desc == ps3.description
```